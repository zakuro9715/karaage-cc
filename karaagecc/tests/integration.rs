#[test]
fn run() {
    assert_eq!(karaagecc::run("10").unwrap().status.code().unwrap(), 10);
    assert_eq!(karaagecc::run("42").unwrap().status.code().unwrap(), 42);
}

#[test]
fn compile_error() {
    assert!(format!("{}", karaagecc::compile("xx").expect_err("")).contains("error"));
}
