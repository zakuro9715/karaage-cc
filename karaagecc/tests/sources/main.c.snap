---
source: karaagecc/tests/integration.rs
expression: "karaage_testutil::run_source(path)"
input_file: karaagecc/tests/sources/main.c
---
stdout_text = '''
10
'''
stdout_bytes = []
stderr_text = ''
stderr_bytes = []
code = 0
